client/src/components/CreateListForm.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function CreateListForm({ name, setName, cancelDisplay, handleSubmit }) {
```

Comment:
An excellent idea! However, I would recommend renaming `cancelDispaly`. I don't think the name really describes what the function does, but more so what the effect for the user is. The function is used to ask the user for a name for the list, a side effect would be that the dialogue is shown.
I know this sounds like nit-picking, but in your `handleSubmit` later `cancelDisplay` will be called and then it's a little unclear what actually happens :)

Reply:
Thanks for your comments. I will change them right away. 👍